custom_components/tuya_local/devices/goodcell_ev_charger.yaml

```diff
@@ -0,0 +1,275 @@
+name: EV charger
+products:
+  - id: j6bzjwhiv2cljjcy
+    name: Goodcell 32A 85-264V 3 phase EV charger
+primary_entity:
+  entity: sensor
+  class: enum
+  name: Status
+  icon: "mdi:ev-station"
+  dps:
+    - id: 101
+      type: string
+      name: sensor
+      mapping:
+        - dps_val: no_connet
+          value: Disconnected
+        - dps_val: connect
+          value: Connected
+        - dps_val: charing
+          value: Charging
+        - dps_val: wait_rfid
+          value: Waiting for authorization
+        - dps_val: finish
+          value: Charged
+        - dps_val: wait_charing
+          value: Delaying charge
+        - dps_val: error
+          value: Error
+    - id: 3
+      type: string
+      optional: true
+      name: charger_status
+      mapping:
+        - dps_val: charger_free
+          value: Standby
+        - dps_val: charger_insert
+          value: Cable connected
+        - dps_val: charger_free_fault
+          value: Charger fault
+        - dps_val: charger_wait
+          value: Charger ready
+        - dps_val: charger_charging
+          value: Charging
+        - dps_val: charger_pause
+          value: Charging paused
+        - dps_val: charger_end
+          value: Charging ended
+        - dps_val: charger_fault
+          value: Charger fault
+    - id: 10
+      type: bitfield
+      name: fault_code
+    - id: 11
+      type: string
+      optional: true
+      name: alarm_set_1
+    - id: 12
+      type: string
+      optional: true
+      name: alarm_set_2
+    - id: 14
```

Comment:
I think this should be a select entity to select the mode of charging.  Otherwise the settings linked to inactive charging modes will be redundant if you cannot switch to that mode.